### PR TITLE
Distance and Strings

### PR DESCRIPTION
**Pull request by ollyau**
- Add C: variables for name and status
- Add new distance calculation
- Adjust XML gauge to reflect C: variable changes

### CODE REVIEW

File: Source/Transformations.h [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#pragma once
+#pragma once
```
Comment (StevenFrost) [inline Source/Transformations.h]: You only need to do this once. I'm sure there's a great joke to be made here but it's 1am.

File: Gauge/SimCache.xml [hunk 2]
```diff
@@ -63,7 +63,7 @@
         </Element>
         <Element id="Element">
             <FloatPosition>186.000,6.000</FloatPosition>
-            <Visibility>(C:SimCache:Distance) 25 &gt; if{ 0 } els{ 1 }</Visibility>
+            <Visibility>(C:SIMCACHE:Distance) 46300 &gt; if{ 0 } els{ 1 }</Visibility>
```
Comment (StevenFrost) [inline Gauge/SimCache.xml:67]: Might be worth adding a comment here, so we know what we're comparing to in sane units :-)

File: Source/Transformations.h (hunk 1 above)
Comment (ollyau) [inline Source/Transformations.h]: Something like `#pragma twice`? :P

**Comment by StevenFrost** [comment]
Approved.

**Pull request merged**